_posts/2010-03-31-peters10a.md

```diff
@@ -5,9 +5,9 @@ abstract: Inferring the causal structure of a set of random variables from a fin
   using additive noise models have been suggested to approach the case of continuous
   variables. In many situations, however, the variables of interest are discrete or
   even have only finitely many states. In this work we extend the notion of additive
-  noise models to these cases. Whenever the joint distribution P^(X,Y) admits such
-  a model in one direction, e.g. Y=f(X)+N, N independent of X, it does not admit the
-  reversed model  X=g(Y)+N’, N’ independent of Y as long as the model is chosen in
+  noise models to these cases. Whenever the joint distribution $\mathbf{P}^{(X,Y)}$ admits such
+  a model in one direction, e.g. $Y = f(X)+N$, $N \perp \!\!\! \perp X$, it does not admit the
+  reversed model  $X=g(Y)+\tilde{N}$, $\tilde{N} \perp \!\!\! \perp Y$ as long as the model is chosen in
```

Comment:
`\perp \!\!\! \perp` is for the symbol of independence. It shows like this on my Google chrome:

<img width="365" alt="Screenshot 2022-02-11 at 5 10 08" src="https://user-images.githubusercontent.com/7121753/153488498-ea253ce3-33fd-4556-b571-233f82404b3a.png">

Ref: https://tex.stackexchange.com/a/455701/263149